app/views/admin/shared/_sidebar.html.slim

```diff
@@ -27,8 +27,6 @@ aside
           = nav_link "Approvals", admin_channel_approvals_path
           = nav_link "Duplicates", duplicates_admin_channels_path
 
-      = nav_link "Organizations", admin_organizations_path
```

Comment:
Can we retain this? It's likely in near future we'll want to expand Organization for all publishers.

Reply:
View code has already been deleted, I think we'd want a slight refactor, and expectations around how this would work 😄